<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWindow</class>
 <widget class="QDialog" name="UserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1920</width>
    <height>1080</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>108</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1920</width>
    <height>1080</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>1931</width>
     <height>971</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab_services">
    <attribute name="title">
     <string>Список услуг</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>1780</width>
       <height>860</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderHighlightSections">
      <bool>false</bool>
     </attribute>
     <attribute name="horizontalHeaderStretchLastSection">
      <bool>false</bool>
     </attribute>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_masters">
    <attribute name="title">
     <string>Список инструкторов</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_masters">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>1780</width>
       <height>860</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_order">
    <attribute name="title">
     <string>Запись</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_schedule">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>1780</width>
       <height>860</height>
      </rect>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::AnyKeyPressed|QAbstractItemView::DoubleClicked|QAbstractItemView::EditKeyPressed</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_reviews">
    <attribute name="title">
     <string>Отзывы</string>
    </attribute>
    <widget class="QTableWidget" name="tableWidget_reviews">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>30</y>
       <width>1311</width>
       <height>860</height>
      </rect>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox">
     <property name="geometry">
      <rect>
       <x>1400</x>
       <y>300</y>
       <width>241</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_review">
     <property name="geometry">
      <rect>
       <x>1400</x>
       <y>20</y>
       <width>491</width>
       <height>261</height>
      </rect>
     </property>
     <property name="dragEnabled">
      <bool>false</bool>
     </property>
     <property name="readOnly">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="pushButton_review">
     <property name="geometry">
      <rect>
       <x>1390</x>
       <y>800</y>
       <width>501</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string>Оставить отзыв</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_like">
     <property name="geometry">
      <rect>
       <x>1480</x>
       <y>530</y>
       <width>361</width>
       <height>271</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_star">
     <property name="geometry">
      <rect>
       <x>1750</x>
       <y>300</y>
       <width>101</width>
       <height>91</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_x">
     <property name="geometry">
      <rect>
       <x>1660</x>
       <y>320</y>
       <width>81</width>
       <height>51</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>X</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="pushButton_exit">
   <property name="geometry">
    <rect>
     <x>1620</x>
     <y>990</y>
     <width>271</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Выйти</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
